notebooks/trial_NGC2992.py

```diff
@@ -14,78 +14,126 @@
 
 from xspec import *
 
-object_name='NGC2992'
+object_name = "NGC2992"
 object_location = swiftbat.simbadlocation(object_name)
-object_batsource = swiftbat.source(ra=object_location[0], dec=object_location[1], name=object_name)
+object_batsource = swiftbat.source(
+    ra=object_location[0], dec=object_location[1], name=object_name
+)
 
-queryargs = dict(time="2004-12-15 .. 2005-12-16", fields='All', resultmax=0)
+queryargs = dict(time="2004-12-15 .. 2005-12-16", fields="All", resultmax=0)
 table_everything = ba.from_heasarc(**queryargs)
 
-minexposure = 1000     # cm^2 after cos adjust
-exposures = np.array([object_batsource.exposure(ra=row['RA'], dec=row['DEC'], roll=row['ROLL_ANGLE'])[0] for row in table_everything])
+minexposure = 1000  # cm^2 after cos adjust
+exposures = np.array(
```

Comment:
This change here is more confusing in my opinion, especially for this example python script.

Reply:
What do you think of the format
```
exposures = np.array(
    [object_batsource.exposure(ra=row["RA"],
                               dec=row["DEC"], 
                               roll=row["ROLL_ANGLE"])[0]
        for row in table_everything
    ])
```
?  (For all three files).  I think it improves the clarity, even if it lacks the purity of letting `black` format everything.